internal/pkg/levee/levee.go

```diff
@@ -78,6 +90,72 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
+// Use the EAR pointer analysis as the propagation engine
+func runEAR(pass *analysis.Pass) (interface{}, error) {
+	conf, _ := config.ReadConfig()
```

Comment:
Prefer passing the config as an argument, instead of reading it again and ignoring the error.